a-seed-from-nothing.sh

```diff
@@ -4,7 +4,7 @@
 SECONDS=0
 
 # Cloud User: cloud-user (CentOS) or ubuntu?
-CLOUD_USER=cloud-user
+CLOUD_USER=ubuntu
```

Comment:
Are you sure we want to change the default to ubuntu?

Reply:
Why not just remove the default user https://github.com/stackhpc/terraform-aufn/pull/16